LiteFarmOrg/LiteFarm
# Feature/fix sensor as standard location

Ticket: [LF-2683](https://lite-farm.atlassian.net/browse/LF-2683)

Summary: 
- Refactors code so that `sensor_id, name, farm_id, deleted` are no longer in the sensor table
- Refactors sensor endpoints so that the URLs are more "RESTful"

To test: 
- Run the latest migrations
- Comment out line 199 in `sensorController` to get rid of webhook error. 
- Test all current sensor flow:
  - Add sensor
  - Go to sensor details page
  - Edit sensor details
  - Sensor readings tab
  - Sensor tasks tab
  - Retire sensor
- If a flow isn't working check first if its working on integration / if there's already a bug ticket for it
